src/foam/u2/filter/properties/ReferenceFilterView.js

```diff
@@ -91,6 +91,9 @@ foam.CLASS({
       background-color: #406dea;
       border-color: #406dea;
     }
+    ^ .p-label {
+      margin-top: 10px;
```

Comment:
Margin doesnt scale well in mobile, prefer to use padding on the parent div instead